src/interfaces/index.ts

```diff
@@ -15,8 +15,9 @@ export interface ProductInfo {
     productPrice: string;
 }
 
-export interface DaysValidInfo {
-    daysValid: string;
+export interface TimePeriodValidInfo {
+    amount: string;
+    timePeriodType: string;
```

Comment:
Could this be something more descriptive, such as `duration`?

Reply:
Which part?

Reply:
Sorry, I was meaning the `timePeriodType` part when I suggested `duration` as an alternative. But I suppose the interface name also feels a bit vague to me